pkgs/by-name/ja/jazz2-content/package.nix

```
@@ -0,0 +1,16 @@
+{ jazz2
+, lib
+, runCommandLocal
+}:
+
+runCommandLocal "jazz2-content"
+{
+  inherit (jazz2) version src;
+
+  meta = jazz2.meta // {
```

Forgot that this now also propagates to over here, sorry.
```suggestion
  meta = (builtins.removeAttrs jazz2.meta ["mainProgram"]) // {
```